website/docs/cluster-management/debugging.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Debugging Clusters
+sidebar_position: 42
+---
+
+Lets work through some common steps when a cluster is not behaving as expected
+
+## Pull request did not appear in github/gitlab
+
+Make sure you're looking at the correct repository
+
+## Cluster does not appear in the UI after merging the PR
+
+- Check that the path the cluster definition was merged to is being reconciled by flux
+- Check that there were no errors in the kustomization resource that applied the cluster definition
+  - There may be a k8s validation error like a bad namespace in the cluster definition
+  - The CAPI provider may not be installed
```

Comment:
How about we link to the documentation that explains _how_ to install CAPI providers?